tools/topology/sof-cml-rt5682-kwd.m4

```diff
@@ -198,7 +198,7 @@ PIPELINE_ADD(sof/pipe-detect.m4,
 	16000, 16000, 16000)
 
 # Connect pipelines together
-SectionGraph."pipe-sof-apl-keyword-detect" {
+SectionGraph."pipe-cml-keyword-detect" {
```

Comment:
so this should not be "pipe-sof-cml-keyword-detect" ? or does it matter?

Reply:
@juimonen it doesnt matter but I fixed it already :d

Reply:
The naming here is not important, but yes, making it be consistent with the file name makes sense to me.